cli/loadkey.go

```diff
@@ -40,7 +40,7 @@ func LoadKey(fm fsdir.T, p ...string) (key ed25519.PrivateKey, err error) {
 			// decoding succeeded, so it's a valid old format key
 			// update the file
 			seed = jsonb.B(dec[:n])
-			err = fm.Set(&seed, p...)
+			err = fm.SetIndented(&seed, p...)
```

Comment:
actually writing keys as "indented" seems funny. it doesn't really make a difference in the output since the value is a string, but i think `Set` would be more appropriate since in this case we don't care about the indentation